.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: go
+
+install:
+  - go get -t -v ./...
+
+script:
+  - go test -race -v ./...
```

Comment:
Actually, for this repo there's so little concurrency that I don't think `-race` is needed. Let's just drop it.